SUTrivBot/Models/Question.cs

```diff
@@ -79,40 +76,30 @@ public AnswerResponse VerifyAnswer(string answer)
                 BonusPoints = BonusPoints
             };
 
+            bool AnswerContains(string item) => answer.Contains(item, StringComparison.InvariantCultureIgnoreCase);
+            
             try
             {
                 if (BonusPoints.HasValue && BonusPoints.Value != 0)
                 {
                     response.BonusPoints = BonusPoints;
                     if (BonusAll ?? false)
                     {
-                        if (BonusAnswers.All(item =>
-                            answer.Contains(item, StringComparison.InvariantCultureIgnoreCase)))
-                            response.AnswerStatus = AnswerStatus.BonusCorrect;
-                        else if (BonusAnswers.Any(item =>
-                            answer.Contains(item, StringComparison.InvariantCultureIgnoreCase)))
-                            response.AnswerStatus = AnswerStatus.PartiallyCorrect;
+                        if (BonusAnswers.All(AnswerContains)) response.AnswerStatus = AnswerStatus.BonusCorrect;
+                        else if (BonusAnswers.Any(AnswerContains)) response.AnswerStatus = AnswerStatus.PartiallyCorrect;

```

Comment:
Why change to no longer ignore case? I don't understand the purpose of this refactor?

Reply:
that was not the purpose of the refactor it is still ignoring case, just centralizing the definition of this lambda

Reply:
`bool AnswerContains(string item) => answer.Contains(item, StringComparison.InvariantCultureIgnoreCase);`

Reply:
My mistake. closing this comment chain